### ISSUE TITLE
[ReactorCraft] Second type of fusion reactor

### ISSUE THREAD

**Issue by nathan67003** [ReikaKalseki/Reika_Mods_Issues#3398]
As discussed over on Discord, posting this here to compile everything into one neat place.

The second reactor type fills the gameplay role of being an intermediary between large fission reactors (and associated safety issues) and the IMMENSE undertaking, both in resources and just making everything fit together and work, that is the Tokamak reactor. It would give less total output (and possibly be less efficient per fusion reaction than the Tokamak to give some incentive towards building the Big Boy) but is simpler to set up and operate as well as more compact.

The type of fusion reactor you settled on for this is some form of inertial confinement fusion (ICF) inspired by the National Ignition Facility. You mentioned envisioning "a single multiblock", "a heat ray array" and "some focusing lenses ultimately converging on a larger - though still not that large or complex - fusion chamber". More quotes from you: "which is fed the two input fluids" "kind of like a preheater"
**Me:** "Or pass water/coolant through it"
**Reika:** "no that abstracts it too much" "it becomes too "all in one"" "hmm" "on a design level i like this idea" "and it does not sound like THAT much work to do" "and then this would have something in the 50GW range or so" "more than a top-tier fission reactor" "but not by a huge marign" "and then if you want hundreds of GW you use a tokamak" "but even 50GW is probably enough for many players" "the simplicity has to come at a cost though" "maybe much lower fuel efficiency compared to a tokamak" "ie it burns far more 2h/3h per unit power"
"i had an idea" "what fuel does the NIF use"
**Me, voidsong:** "DT fuel pellets", "DT slush"
**Reika:** "i might well ignore the solid fuel aspect of it" "since fluid fed reactors are more interesting than item transport"
**Me:** "Hell, you could treat the solids as a fluid" "If the pellets are small enough, the simplification kinda works"
**Reika:** "does it have to be cryogenic?" "if so that could add something interesting" "though the crystallizer is only for item production" "not fluids"
**voidsong:** "well, it needs to not boil off or break the tiny tiny holraum"
**Me:** "Not necessarily, but it certainly is IRL" "Since the pellets are solid hydrogen"
**Reika:** "hmm" "ok" "maybe i will keep that" "let me write this down" "one problem" "crystallizer has one input fluid"
[omitted, mostly unrelated discussion + unfructuous brainstorming]
**Me:** "So what about a whack of the screwdriver/appropriate tool that'd set the irradiation chamber to output a DT mix instead of pure tritium?" "@ Reika, how about using the neutron irradiation chamber with an alternative operating mode? Instead of being able to turn deuterium into pure tritium, it'd turn it into a DT mix"
[brief back-and-forth where you assume I'm talking about having multiple outputs and I attempt to explain better that it's a single DT mix output]
**Reika:** "oh I see" "see now THAT is viable"

**In short:** a single multiblock consisting of a heat ray array, focusing lenses and a fusion chamber; the fusion reaction outputs heat which can either be extracted directly with boilers or by running a coolant of some sort (liquid sodium heat exhanger?) through the multiblock. The multiblock is fed fuel in the form of a cryogenic mixture of DT slurry; the slurry is (MAYBE) made in the crystallizer (or whatever is used to make liquid nitrogen) by feeding it a "low temperature" DT mix; the DT mix is made in a neutron irradiation chamber set to input deuterium and output a DT mix.
